R/grants.R

```diff
@@ -39,4 +39,7 @@
 #' vector of data points used for training}
 #' @source Kuhn and Johnson (2013). _Applied Predictive Modeling_. Springer.
 #' @keywords datasets
+#' @examples
+#' data(grants)
+#' str(grants_other)
```

Comment:
Can you add `grants_test` and `grants_2008` as well?

Reply:
👍